content/en/docs/contribute/participate/roles-and-responsibilities.md

```diff
@@ -32,7 +32,7 @@ Anyone can:
 - Suggest improvements on [Slack](https://slack.k8s.io/) or the
   [SIG docs mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-docs).
 
-After [signing the CLA](/docs/contribute/new-content/overview/#sign-the-cla), anyone can also:
+After [signing the CLA](https://github.com/kubernetes/community/blob/master/CLA.md), anyone can also:
```

Comment:
I think the `git.k8s.io` short link would be better.

```suggestion
After [signing the CLA](https://git.k8s.io/community/CLA.md), anyone can also:
```

Reply:
Short links are good for someone, but they may not **work** for everyone.